src/components/Categorizer.js

```diff
@@ -98,26 +95,12 @@ class Categorizer extends React.Component {
 }
 
 Categorizer.propTypes = {
-  currentUserState: PropTypes.shape({
-    transactions: PropTypes.shape({
-      uniquePartiesList: PropTypes.arrayOf(
-        PropTypes.shape({
-          date: PropTypes.string.isRequired,
-          amount: PropTypes.string.isRequired,
-          party: PropTypes.string.isRequired
-        })
-      )
-    }),
+  userState: PropTypes.shape({
+    uniqueParties: PropTypes.arrayOf(PropTypes.string).isRequired,
     categories: PropTypes.arrayOf(
       PropTypes.shape({
-        categoryTitle: PropTypes.string,
-        categoryParties: PropTypes.arrayOf(
-          PropTypes.shape({
-            date: PropTypes.string,
-            amount: PropTypes.string,
-            party: PropTypes.string.isRequired
-          })
-        )
+        title: PropTypes.string,
+        parties: PropTypes.arrayOf(PropTypes.string)
```

Comment:
I wonder if it could cause potential issues if we use the transaction's title as an ID to identify unique parties. Can you think of any?

Reply:
I don't know 🤔 I have to look up if we possibly could use some IDs specific to a party from the CSVs. But then we have a problem if some banks have it, and some don't